intg-androidtv/config.py

```diff
@@ -250,6 +252,9 @@ def migration_required(self) -> bool:
 
     async def migrate(self) -> bool:
         """Migrate configuration if required."""
+        # pylint: disable=C0415,R0401
+        from tv import AndroidTv
```

Comment:
To avoid this hack and cyclic import - if we keep the `AtvDevice` usage in `AndroidTv` -  the `migrate` method should be refactored as an external function in a separate file.

We can keep it for the moment and focus on the main functionality, even though I really don't like cyclic imports. It's usually only a matter of time until the "design" doesn't work anymore and requires more ugly hacks.